support-frontend/assets/helpers/internationalisation/currency.ts

```diff
@@ -137,15 +137,8 @@ const spokenCurrencies: Record<IsoCurrency, SpokenCurrency> = {
 };
 
 // ----- Functions ----- //
-function fromCountryGroupId(
-	countryGroupId: CountryGroupId,
-): IsoCurrency | null | undefined {
-	const countryGroup: CountryGroup | null | undefined =
-		countryGroups[countryGroupId];
-
-	if (!countryGroup) {
-		return null;
-	}
+function fromCountryGroupId(countryGroupId: CountryGroupId): IsoCurrency {
+	const countryGroup = countryGroups[countryGroupId];
```

Comment:
This one makes me slightly nervous. Given the type signature, `countryGroup` is always defined so we don't need the conditional. Typescript should catch anywhere that we use it incorrectly but I'm still worried that there might be some code that depends on it

Reply:
I looked for every instance of type assertions casting to `CountryGroupId` and they all look fine so this should be safe